### PR TITLE
Add mobile support

### PR DESCRIPTION
**Pull request by TearGosling**
Taken from Kobold's colab

### CODE REVIEW

File: notebooks/GPU.ipynb [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+        "<audio src=\"https://henk.tech/colabkobold/silence.m4a\" controls>"
+      ],
       "metadata": {
-        "id": "view-in-github",
-        "colab_type": "text"
+        "id": "k9hM83vkDfZ4"
       },
-      "source": [
-        "<a href=\"https://colab.research.google.com/github/PygmalionAI/gradio-ui/blob/master/notebooks/GPU.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
-      ]
```
Comment (0x000011b) [inline notebooks/GPU.ipynb]: Any reason to remove the "Open in Colab" link? It's useful to people who see the notebook in GitHub first.

File: notebooks/GPU.ipynb [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
   "cells": [
     {
-      "cell_type": "markdown",
+      "cell_type": "code",
+      "source": [
+        "# Taken from KoboldAI's Colab.\n",
+        "# If it ain't broke don't fix it lol\n",
+        "#@title <-- Tap this if you play on Mobile { display-mode: \"form\" }\n",
+        "%%html\n",
+        "<b>Press play on the music player to keep the tab alive, then start KoboldAI below (Uses only 13MB of data)</b><br/>\n",
```
Comment (0x000011b) [inline notebooks/GPU.ipynb]: Maybe consider removing the "start KoboldAI below" mention? Might confuse some people, since the inference server being KAI is just an internal implementation detail.

**Review by 0x000011b — CHANGES_REQUESTED** [review]

File: notebooks/GPU.ipynb (hunk 1 above)
Comment (TearGosling) [reply notebooks/GPU.ipynb]: Whoops. Must've not seen that. Not meant to be removed.

File: notebooks/GPU.ipynb (hunk 2 above)
Comment (TearGosling) [reply notebooks/GPU.ipynb]: Huh. Yeah, I probably should do that too. I figured since it's not actually visible on Colab, no one will see it, but it probably does show up in some way.

File: notebooks/GPU.ipynb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "cells": [
-    {
+	{
```
Comment (0x000011b) [inline notebooks/GPU.ipynb]: Minor nitpick: unnecessary whitespace change

File: notebooks/GPU.ipynb [hunk 4]
```diff
@@ -84,4 +100,4 @@
   },
   "nbformat": 4,
   "nbformat_minor": 0
-}
+}
```
Comment (0x000011b) [inline notebooks/GPU.ipynb]: Minor nitpick: unnecessary whitespace change

File: notebooks/GPU.ipynb (hunk 3 above)
Comment (TearGosling) [reply notebooks/GPU.ipynb]: Fuck, I hate whitespace changes. Four spaces instead of tab?

Comment (0x000011b) [reply notebooks/GPU.ipynb]: Yep, it was originally 4 spaces and seems to have been replaced by a tab character.

**Review by 0x000011b — APPROVED** [review]
LGTM!
